app/assets/stylesheets/moderators.scss

```diff
@@ -213,36 +213,205 @@
     margin: 90px auto;
     border-radius: 12px;
   }
+
+  // Reaction Buttons START
+
+  .thumb-reactions {
+    display: flex;
+    justify-content: center;
+  }
+
   .reaction-button {
     position: relative;
-    width: 80px;
-    height: 80px;
     background: white;
     border-color: white;
-    margin: 30px 30px;
-    border-radius: 200px;
-    border: 0px;
-    padding: 12px 12px;
+    border: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 156px;
```

Comment:
An alternative way to setting the position of the two `reaction-button` element would be the grid layout. Doing this would mean not needing to declare a width value for the button because it would be based on the parent element's padding. Here is an illustration of how you can do it: https://codepen.io/lisasy/pen/xxwgOEO